scripts/zkai_attention_kv_stwo_native_d128_two_head_seq32_bounded_softmax_table_proof_input.py

```
@@ -0,0 +1,1020 @@
+                ),
+                "new_value": shifted_vector(
+                    base["new_value"],
+                    width=VALUE_WIDTH,
+                    head_index=head_index,
+                    local_step=local_step,
+                    role_bias=3,
+                    label=f"input_steps[{len(extended)}].new_value",
+                ),
+            })
+    if len(extended) != HEAD_COUNT * SEQUENCE_LENGTH:
+        raise AttentionKvD128TwoHeadSeq32BoundedSoftmaxTableInputError("extended input step length drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)